test/SourceKit/CursorInfo/Inputs/Swift.swift

```diff
@@ -0,0 +1,4 @@
+// This is built with '-parse-stdlib', so we can't use any stdlib types.
```

Comment:
Could you use `split-file` instead 🙏 I find it makes these so much easier to follow outside of a review (ie. when trying to work out what a test does after the fact).